Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -461,6 +461,8 @@ lemma IsCycle.isPath_takeUntil {c : G.Walk v v} (hc : c.IsCycle) (h : w ∈ c.su
   rw [← isCycle_reverse, ← take_spec c h, reverse_append] at hc
   exact (c.takeUntil w h).isPath_reverse_iff.mp (hc.isPath_of_append_right (not_nil_of_ne hvw))
 
+/-- Taking a strict initial segment of a path removes the end vertex from the support.
+TODO : a `dropUntil` version. -/
```

Comment:
I think we should be judicious with `TODO`s: we could easily stuff the library with them and it will just create clutter and a slight maintenance burden.

OTOH they are sometimes a good idea. In this case I'm dubious but I'll let you decide.
```suggestion
/-- Taking a strict initial segment of a path removes the end vertex from the support. -/
```